aspnetcore/host-and-deploy/docker/visual-studio-tools-for-docker.md

```diff
@@ -103,9 +103,9 @@ baf9a678c88d        hellodockertools:dev   "C:\\remote_debugge..."   21 seconds
 
 ## Edit and continue
 
-Changes to static files and Razor views are automatically updated without the need for a compilation step. Make the change, save, and refresh the browser to view the update.  
+Changes to static files and Razor views are automatically updated without the need for a compilation step. Make the change, save, and refresh the browser to view the update.
 
-Modifications to code files requires compiling and a restart of Kestrel within the container. After making the change, use CTRL + F5 to perform the process and start the app within the container. The Docker container isn't rebuilt or stopped. Run the `docker ps` command in PMC. Notice the original container is still running as of 10 minutes ago:
+Code file modifications require compilation and a restart of Kestrel within the container. After making the change, use CTRL + F5 to perform the process and start the app within the container. The Docker container isn't rebuilt or stopped. Run the `docker ps` command in PMC. Notice the original container is still running as of 10 minutes ago:
```

Comment:
They're calling for ordinary capitalization (`Ctrl`). Are they calling for a space, too? I think either `Ctrl+F5` or `Ctrl + F5`.